pre-commit.sh

```
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Running check and validate scripts in commits.
+# Validates Typescript compilation, formatting, linting
+# test cases and attempts prod build.
+BRANCH_NAME=$(git branch | grep '*' | sed 's/* //')
```

Shellcheck (0.10.0)

[warning] Grep uses regex, but this looks like a glob.

(SC2063)